src/applications/vaos/appointment-list/pages/PastAppointmentsPage/index.jsx

```diff
@@ -211,17 +215,11 @@ export default function PastAppointmentsPage() {
 
       {!keys.length && (
         <div className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-y--3">
```

Comment:
You can remove the class "vads-u-margin-y--3" from above.  It doesn't seem to do anything.